contribs/gmf/src/directives/objectediting.js

```diff
@@ -527,13 +527,22 @@ gmf.ObjecteditingController.prototype.handleDeleteFeature_ = function(resp) {
  * @private
  */
 gmf.ObjecteditingController.prototype.handleEditFeature_ = function(resp) {
+  // (1) Update the id
+  var features = new ol.format.GeoJSON().readFeatures(resp.data);
```

Comment:
Fixed.  Do you know why `make dist` didn't complain when we're missing a require?